plone/app/caching/browser/controlpanel.py

```diff
@@ -344,6 +344,11 @@ def processSave(self):
             ramCacheCleanupInterval,
         )
 
+        IStatusMessage(self.request).addStatusMessage(
+            _(u'Purging ist still enabled while caching is disabled!'),
+            'warn',
```

Comment:
```suggestion
            'warning',
```
use warning to make Plone styles work